include/mbedtls/ssl_internal.h

```diff
@@ -1301,4 +1334,45 @@ void mbedtls_ssl_buffering_free( mbedtls_ssl_context *ssl );
 void mbedtls_ssl_flight_free( mbedtls_ssl_flight_item *flight );
 #endif /* MBEDTLS_SSL_PROTO_DTLS */
 
+/* Checks whether client handshake defragmentation is enabled */
+static inline int mbedtls_ssl_hs_defrag_enabled( const mbedtls_ssl_context *ssl )
+{
+#if defined(MBEDTLS_SSL_HS_DEFRAG_CLT)
+    if( ( ssl->handshake ) &&
+        ( 0 < ssl->handshake->defrag.acc_len ) &&
+        ( ssl->conf->endpoint == MBEDTLS_SSL_IS_CLIENT ) &&
```

Comment:
As above: Is there any benefit of restricting to the client here? It seems we'd get the feature on the server for free.